.gitignore

```
@@ -0,0 +1,3 @@
+node_modules
+dist
```

Всё-таки попали файлы из `dist`, надо исключить. Как вариант можно попробовать заменить на такого вида строку
```suggestion
dist/
```

---

Вообще ощущение, что сначала файлы из `dist` улетели в git, а уже потом был заполнен `.gitignore`. Тогда нужно удалить файлы страниц